src/aws/osml/models/server_utils.py

```diff
@@ -104,3 +110,41 @@ def detect_to_geojson_dict(
         },
         "type": "Feature",
     }
+
+    if fixed_object_mask:
+        geojson["polygon_imcoords"] = (mask_to_polygon(fixed_object_mask),)
```

Comment:
I thought we decided to make this "geometry_imcoords" and follow the structure of the [GeoJSON geometry object](https://datatracker.ietf.org/doc/html/rfc7946#section-3.1). The only difference is that this property will have coordinates will in (x, y) pixels instead of latitude and longitude. Beyond general consistency this will let us easily expand to Point and LineString  detections if a model chooses to return that type of feature.   

`
"geometry_imcoords": {
               "type": "Polygon",
               "coordinates": [
                   [
                       [100, 0],
                       [101, 0],
                       [101, 1],
                       [100, 1],
                       [100, 0]
                   ]
               ]
           },
`

Reply:
I was tracking (per our [design quip](https://quip-amazon.com/4qGcAznFnCuZ/AWS-OSML-Overhead-Inference-Pipeline-Segmentation-Support)) that we were going to use geometry_imcoords for the 'properties' - then the GeoJSON geometry would be used for the world-coordinate block. 
that's why this is as it is. 
If we change it, that's fine - I'll just note where we land, and make sure our NC GIS inference container follows wherever this lands.

Reply:
We shouldn't just add new fields to a GeoJSON feature. The "geometry_imcoords" inside the properties replaces this "polygon_imcoords" which never should have been outside of the properties structure.

Reply:
I will update this name accordingly - good catch - I forgot about that bit